### ISSUE TITLE
Audio play and loop

### ISSUE THREAD

**Issue by Ilohama** [TheCherno/Sparky#26]
There's another issue (not sure that's important, because this is for Game but....). Let's try to play a sound and try to loop the same sound (with play() function and loop() function on the same sound).... The loop() stop at the end of play().

Have a nice day~~

**Comment by vmarkushin** [comment]
Because loop() is like: play() sound every time it ends. And play() is like: play() once.

**Comment by Ilohama** [comment]
yeah i know that, i just wanted to report the issue, can be a bug or not whatever.
